language/stdlib/modules/libra_account.move

```diff
@@ -82,16 +82,37 @@ module LibraAccount {
     // A privilege to allow the freezing of accounts.
     struct FreezingPrivilege { }
 
+    // Message for freeze account events
+    struct FreezeAccountEvent {
+        // The address that initiated freeze txn
+        initiator_address: address,
+        // The address that was frozen
+        frozen_address: address,
+    }
+
+
+    // Message for freeze account events
+    struct UnfreezeAccountEvent {
+        // The address that initiated unfreeze txn
+        initiator_address: address,
+        // The address that was unfrozen
+        unfrozen_address: address,
+    }
+
     resource struct AccountOperationsCapability {
         tracking_cap: AccountTrack::CallingCapability,
         event_creation_cap: Event::EventHandleGeneratorCreationCapability,
+        freeze_event_handle: Event::EventHandle<FreezeAccountEvent>,
+        unfreeze_event_handle: Event::EventHandle<UnfreezeAccountEvent>,
```

Comment:
Question for @sblackshear @tzakian, should we have the two handles in the `AccountOperationsCapability` or a separate resource? I can see arguments for both.

Reply:
Good question, and I think either way would be fine. That being said, I personally think having it in this resource is better since this holds all of the singleton resources needed for privileged account operations in one place.